packages/utilities/psammead-storybook-helpers/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 5.0.0 | [PR#1714](https://github.com/bbc/psammead/pull/1714) Renamed `indonesian` to `indonesia` and `afaanOromoo` to `afaanoromoo`.  Input provider now has an optional 4th argument that allows the default service to be configured |
```

Comment:
Any reason you went with `5.0.0` rather than `4.1.0`, IMO this doesn't need to be a major change

Reply:
I believe the service rename part of this is a breaking change - previously, "indonesian" appeared on the dropdown, and got passed to your component if someone selected it.  Now, it will be "indonesia"

At least I think that's true - the only thing that makes me doubt it is surely some components would have errored if you passed an invalid service name in the current version?

But maybe nobody noticed it as these services weren't used

Reply:
Ahhh yeah, it is likely those services haven't been used/bugs haven't been spotted.